website/source/docs/guides/servers.html.md

```diff
@@ -120,7 +163,7 @@ To remove all agents that accidentally joined the wrong set of servers, clear ou
 
```

Comment:
I'm not the author of this statement, so I'm not sure what it means. I would actually like to remove it since it seems like a tangent. Thoughts about removing it @banks

Reply:
I think it's worth saying something just because every other option here assumes the cluster is healthy to start with. But maybe we could replace it with:

> These graceful methods to add or remove servers assume you have a healthy cluster. If the cluster has no leader due to loss of quorum or data corruption please see the  [outage recovery guide](/docs/guides/outage.html#manual-recovery-using-peers-json).

Reply:
Updated!